LayoutTests/fast/canvas/font-family-system-ui-canvas-expected-mismatch.html

```diff
@@ -0,0 +1,8 @@
+<canvas id="myCanvas" width="200" height="100"></canvas>
+
+<script>
+var c = document.getElementById("myCanvas");
+var ctx = c.getContext("2d");
+ctx.font = "30px times";
```

Comment:
`serif` might be better than `times` fwiw.

Reply:
Changed